edb/server/defines.py

```diff
@@ -26,6 +26,9 @@
 EDGEDB_ENCODING = 'utf-8'
 EDGEDB_VISIBLE_METADATA_PREFIX = r'EdgeDB metadata follows, do not modify.\n'
 
+# Increment this whenever the database layout or stdlib changes.
+EDGEDB_CATALOG_VERSION = 20191127
```

Comment:
I'd increase the resolution and add hours/minutes.

```suggestion
EDGEDB_CATALOG_VERSION = 2019_11_27_00_00
```

Reply:
(in case a few incompatible changes are made within one day)